File: EjemploCodigo/Python/SubClasesPayment/Card.py
from Payment import Payment

class Card(Payment):
    number     = int
    fechaVec   = str
    cvv        = str
    
    def __init__(self, id, fecha, number, cvv, fechaVec):
        super().__init__(id, fecha)
        self.number    = number
        self.fechaVec  = fechaVec 
        self.cvv       = cvv 

File: EjemploCodigo/PHP/SubClasesPayment/Card.php
<?php
require_once('../Payment.php');

class Card extends Payment{
    public $numberCard;
    public $dateVec;
    public $cvv; 

    public function __construct($id, $fecha, $numberCard, $dateVec, $cvv ) {
        parent ::__construct($id, $fecha); //llamo al metodo consturctor de Payment ->  esto es por que Payment tiene un constructor 
        $this->numberCard = $numberCard;
        $this->dateVec = $dateVec;
        $this->cvv = $cvv;
    }

}
